public/app/features/provisioning/RepositoryListPage.tsx

```diff
@@ -150,8 +150,8 @@ function getListURL(repo: Repository, stats: ResourceCount): string {
   if (stats.resource === 'playlists') {
     return '/playlists';
   }
-  if (repo.spec?.folder) {
-    return `/dashboards/f/${repo.spec?.folder}`;
+  if (repo.spec?.sync.target === 'mirror') {
```

Comment:
shouldn't we use some constant from a common place for this one?

Reply:
likely!  but rtk does not make them for us 😬